x/orderbook/amount.go

```diff
@@ -0,0 +1,16 @@
+package orderbook
+
+
+func NewAmount(whole int64, fractional int64) Amount {
+	return Amount{
+		Whole: whole,
+		Fractional: fractional,
+	}
+}
+
+func NewAmountp(whole int64, fractional int64) *Amount {
+	return &Amount{
+		Whole: whole,
+		Fractional: fractional,
+	}
+}
```

Comment:
Small detail but new line at the end of the file is missing :blush:

Reply:
I should add an .editorconfig file here. Good eye